app/views/recommended_links/_update_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for(recommended_link) do |f| %>
```

Comment:
can you merge `_create_form` with this form ? You could try to see if `object.new_record?` (http://apidock.com/rails/ActiveRecord/Base/new_record%3F)